src/appleseed-max-impl/appleseedrenderer/renderersettings.cpp

```diff
@@ -110,7 +111,7 @@ void RendererSettings::apply_common_settings(asr::Project& project, const char*
 
     params.insert_path("pt.enable_ibl", m_background_emits_light);
     params.insert_path("pt.enable_caustics", m_caustics);
-
+	params.insert_path("use_embree", m_use_embree);
```

Comment:
That line looks good but it's in the middle of the PT settings. Move it somewhere else, maybe right before or after setting `sampling_mode`?